scripts/ci/common.sh

```diff
@@ -14,6 +14,14 @@ function runTests() {
   make ${targets}
 }
 
+if [ ! -z "${APM_SERVER_BRANCH}" ]; then
+  export BUILD_OPTS="${BUILD_OPTS} --apm-server-build https://github.com/elastic/apm-server.git@${APM_SERVER_BRANCH}"
```

Comment:
do all of these `BUILD_OPTS` need to be exported?  I don't they do

Reply:
yep, it is not needed.